content/en/guides/hosting/iam/scim.md

```diff
@@ -1173,6 +1424,128 @@ POST /scim/Roles
 }
 ```
 
+### Update custom role
+
+#### Add permissions to role
+
+- **Endpoint**: **`<host-url>/scim/Roles/{id}`**
+- **Method**: PATCH
+- **Description**: Add permissions to an existing custom role.
```

Comment:
Where are the possible permissions listed? Do we want to show how to list the custom role's current permissions first? What about creating a custom role?